assets/prototype/domain/eventEditor/datesList/AddNewDateButton.js

```diff
@@ -1,21 +1,40 @@
-import { Button } from '@blueprintjs/core/lib/esm';
-import { AppToaster } from '../EventEditor';
+import { useState } from "@wordpress/element";
+import { Button } from "@blueprintjs/core/lib/esm";
+import AddNewDateModal from "./AddNewDateModal";
 
 const btnStyle = {
-	margin: '0 0 0 1rem',
+	margin: "0 0 0 1rem"
```

Comment:
linting rules require single quotes no?
or was that part of `@wordpress/eslint-plugin` ?

Reply:
Updated `.prettierrc`, thx.